### ISSUE TITLE
shared private folder timeout.

### ISSUE THREAD

**Issue by grublets** [keybase/client#3443]
This occurs when trying anything within a shared private folder. My own private folder is fine.

my log id: 24686ced9e2ced9a72b0b51c

**Comment by grublets** [comment]
update: 

occurs on MacOS - Darwin weiner.local 15.5.0 Darwin Kernel Version 15.5.0: Tue Apr 19 18:36:36 PDT 2016; root:xnu-3248.50.21~8/RELEASE_X86_64 x86_64

Works fine on Linux Mint - Linux crud 4.4.0-21-generic #37-Ubuntu SMP Mon Apr 18 18:33:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**Comment by maxtaco** [comment]
We dot support macOS beta yet. We are working on it!

**Comment by cjb** [comment]
@maxtaco (I might be wrong, but I don't see anything suggesting this is using the beta)

**Comment by strib** [comment]
Yeah it's not on the beta.  From the updater log: `OSVersion:"10.11.5"`.

This is another timeout in running identifies.  See the identify starting at `2016-07-13T07:13:21.679467` in the service log.

**Comment by maxtaco** [comment]
Oh sorry whoops.
